connectors/bitcoin.go

```diff
@@ -817,3 +818,18 @@ func getOpCodeFromInt(val int) byte {
 		return txscript.OP_16
 	}
 }
+
+func isP2PKH(address string) bool {
+	pattern := regexp.MustCompile(`^[13][a-km-zA-HJ-NP-Z0-9]{25,34}$`)
+	return pattern.MatchString(address)
+}
+
+func isP2SH(address string) bool {
+	pattern := regexp.MustCompile(`^3[a-km-zA-HJ-NP-Z0-9]{25,34}$`)
```

Comment:
How about p2sh for test env? starting with 2?

Reply:
@MaximStanciu8 case addressed